config/jobs/kubernetes-sigs/cluster-api/cluster-api-periodics-main.yaml

```
@@ -47,7 +47,7 @@ periodics:
           - name: GINKGO_FOCUS
             value: "\\[clusterctl-Upgrade\\]"
           - name: INIT_WITH_BINARY
-            value: https://github.com/kubernetes-sigs/cluster-api/releases/download/v0.3.23/clusterctl-{OS}-{ARCH}
+            value: https://github.com/kubernetes-sigs/cluster-api/releases/download/v0.3.26/clusterctl-{OS}-{ARCH}
```

```suggestion
            value: https://github.com/kubernetes-sigs/cluster-api/releases/download/v0.3.25/clusterctl-{OS}-{ARCH}
```

If I got it correctly v0.3.25 is not affected, so there will be no v0.3.26

---

Same below